e2e/bitrise.yml

```diff
@@ -373,24 +373,24 @@ workflows:
         title: Start a failing workflow, wrapped in a script.
         run_if: '{{enveq "XCODE_MAJOR_VERSION_BETWEEN_11_AND_12" "true"}}'
         inputs:
-          - content: |-
-              #!/bin/env bash
-              set -x # Do not set -e as bitrise command is expected to fail
-              bitrise run --config=./e2e/bitrise.yml utility_test_no_retry_on_failure
-              if [ $? -ne 1 ] ; then
-                echo "Workflow was expected to fail, exit code not 1."
-                exit 1
-              fi
+        - content: |-
+            #!/bin/env bash
+            set -x # Do not set -e as bitrise command is expected to fail
+            bitrise run --config=./e2e/bitrise.yml utility_test_should_retry_test_on_fail_off
+            if [ $? -ne 1 ] ; then
+              echo "Workflow was expected to fail, exit code not 1."
+              exit 1
+            fi
 
-  utility_test_no_retry_on_failure:
+  utility_test_should_retry_test_on_fail_off:
     envs:
-    - TEST_APP_FLAKY_TEST_NUMBER_OF_FAILURES: 1
+    - TEST_APP_NUMBER_OF_INITIAL_TEST_FAILURES: 1
     - EXPECT_TEST_FAILURE: "true"
     - TEST_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
-    - TEST_APP_BRANCH: master
+    - TEST_APP_BRANCH: STEP-1203-split-flaky-test-cases
```

Comment:
Should be updated everywhere once https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test/pull/7 is merged.